pkg/policies/scorecard/scorecard.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const configFile = "scorecard.yaml"
-const polName = "Security Scorecards"
+const polName = "OpenSSF Scorecard"
```

Comment:
@jeffmendoza — is this breaking? It's local, so I'm guessing "no", but just wanted to flag it on the review.

Reply:
So this will change the name needed for the `-policy` cli option to only run this policy. https://github.com/ossf/allstar/blob/main/cmd/allstar/main.go#L46-L62

It will change the "area" value for all the structured logging in this policy.

It will change the name of the policy in the structured summary results that allstar logs on each loop https://github.com/ossf/allstar/blob/main/pkg/enforce/enforce.go#L157-L160

I think this is ok and is just something to call out in release notes.

Reply:
Thanks for the pointers, @jeffmendoza!
I've filed https://github.com/ossf/allstar/issues/537 to track this.